### PR TITLE
Enable win_arm64

### PR DESCRIPTION
**Pull request by isuruf**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by isuruf** [comment]
@conda-forge-admin, rerender

**Comment by isuruf** [comment]
@conda-forge/bzip2, this is ready

**Comment by isuruf** [comment]
@conda-forge/core, mind merging this? I added myself to maintainers.

**Comment by isuruf** [comment]
@conda-forge-admin, rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -70,5 +70,5 @@ extra:
     - jakirkham
     - pelson
     - jlanga
-    - mingwandroid
```
Comment (hmaarrfk) [inline recipe/meta.yaml:73]: :(

**Review by hmaarrfk — APPROVED** [review]
was there really no change in the build script needed? interesting....

**Comment by isuruf** [comment]
Yes, lucky I guess.

**Comment by hmaarrfk** [comment]
do you have a native windows arm machine to test things on? 

Either way, merge at will.

**Comment by isuruf** [comment]
Yes, got a windows arm box from Microsoft.

**Pull request merged**